### PR TITLE
slice declaration: nil better than empty

### PR DESCRIPTION
**Pull request by lilongfeng0902**
slice declaration：
`var fields []string` is better than `fields := []string{}`

### CODE REVIEW

**Comment by medyagh** [comment]
@lilongfeng0902 do u mind explaining why this is better?

**Comment by lilongfeng0902** [comment]
> @lilongfeng0902 do u mind explaining why this is better? and what problem are you trying to solve?

@medyagh, thamks for your review. This PR is just coding review, the point that is modified may be better. You may find losts of commit histories, the commits are related with [https://github.com/kubernetes/minikube/pull/11196](url). Now let me explain why this modification is better：
First, I compare the slice declaration with others, most programmers use the style which I proposal.
Second, the style of `fields := []string{}` is prompted `empty slice declaration via literal`, not mistake, just suggestion.
Third, the style of `fields := []string{}` may be allocated space, the style of `var fields []string` is just declared.
In summary, my explanation may not be reasonable, then I will make other efforts for  learning minikube. Thank you again.

**Comment by lilongfeng0902** [comment]
@medyagh btw, you may find that thare are five commit histories, but there is only the last one related with PR. How could I deal with this situation?

**Comment by ilya-zuyev** [comment]
Go compiler generates the same code for both options. IMO, it's only matter of style

**Comment by lilongfeng0902** [comment]
> Go compiler generates the same code for both options. IMO, it's only matter of style

Thanks for your attention and review,  from your point and medyagh's doubt, this PR is unreasonable and no value, I will avoid this in the future. Thank you again.
